### PR TITLE
Finishing setting up API

### PR DESCRIPTION
**Pull request by stephenJohnGiardina**
@asmit222 Hey. Can you review this? I'm just fixing some stuff so that my API works.

### CODE REVIEW

**Comment by stephenJohnGiardina** [comment]
@marcusbarnesdeveloper Can you review my PR please?

**Comment by stephenJohnGiardina** [comment]
@geophray Can you review my PR here?

File: client/components/ItemName.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    }
+  }
+
+  render() {
+    return (
+      <div style={{fontSize: 25}}>
```
Comment (geophray) [inline client/components/ItemName.jsx:16]: Is this styling rendering as you expect? I am unfamiliar with this syntax.

File: client/components/ItemDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <ul>
+          {this.props.details.map((item, key) => {
+            return (<li key={key}>{item}</li>)
+          })}
```
Comment (geophray) [inline client/components/ItemDetails.jsx:17]: Can you pass in two parameters to a map function like this?

**Comment by geophray** [comment]
Sorry for the delay on getting this reviewed.

File: client/components/ItemName.jsx (hunk 1 above)
Comment (stephenJohnGiardina) [reply client/components/ItemName.jsx:16]: Yes. This is working for now. This is inline styling in react.

File: client/components/ItemDetails.jsx (hunk 2 above)
Comment (stephenJohnGiardina) [reply client/components/ItemDetails.jsx:17]: Yes. Map has an optional second parameter which is the index of the item in the array that is being mapped.

Comment (stephenJohnGiardina) [reply client/components/ItemDetails.jsx:17]: It also has a third optional parameter which is the entire array.

File: client/components/ItemName.jsx (hunk 1 above)
Comment (geophray) [reply client/components/ItemName.jsx:16]: Gotcha. Hadn't gotten around to that yet with my own module.

File: client/components/ItemDetails.jsx (hunk 2 above)
Comment (geophray) [reply client/components/ItemDetails.jsx:17]: Sure enough. I feel kind of silly now. :)

**Pull request merged**